examples/cifar10/pt/utils/cifar10_download_data.py

```diff
@@ -41,7 +41,7 @@
 
 
 def main():
-    splitter = Cifar10DataSplitter()
+    splitter = Cifar10DataSplitter(custom_CIFAR10_ROOT="/persistenttmp/cifar10")
```

Comment:
instead of modifying codes here, it would be better if CI can do its own data preparation.